app/ts/types/interceptor-messages.ts

```diff
@@ -687,6 +712,38 @@ export const ChangeAddOrModifyAddressWindowState = funtypes.ReadonlyObject({
 	})
 })
 
+export type RetrieveWebsiteAccessFilter = funtypes.Static<typeof RetrieveWebsiteAccessFilter>
+export const RetrieveWebsiteAccessFilter = funtypes.ReadonlyObject({
+	query: funtypes.String,
+}).asReadonly()
+
+export type RetrieveWebsiteAccess = funtypes.Static<typeof RetrieveWebsiteAccess>
+export const RetrieveWebsiteAccess = funtypes.ReadonlyObject({
+	method: funtypes.Literal('popup_retrieveWebsiteAccess'),
+	data: RetrieveWebsiteAccessFilter,
+}).asReadonly()
+
+export type SearchProximity = funtypes.Static<typeof SearchProximity>
+export const SearchProximity = funtypes.ReadonlyTuple(funtypes.Number, funtypes.Number)
+
+export type SearchMetadata = funtypes.Static<typeof SearchMetadata>
+export const SearchMetadata = funtypes.Object({
+	_targets: funtypes.MutableArray(funtypes.String),
```

Comment:
why _ ? and mutable array?

Reply:
search will be reintroduced on https://github.com/DarkFlorist/TheInterceptor/pull/1121